src/components/token/TokenAutoCompleteField.js

```diff
@@ -56,12 +57,36 @@ class TokenAutoCompleteField extends React.Component {
 
   componentDidMount() {
     document.addEventListener('click', this.handleClick);
+
+    if (this.props.tokenId !== hathorLibConstants.HATHOR_TOKEN_CONFIG.uid) {
+      // A token was selected in the query params
+      // so we must search for it here to add
+      // in the autocomplete input and perform the search
+      this.searchAndSelectToken(this.props.tokenId);
+    }
   }
 
   componentWillUnmount() {
     document.removeEventListener('click', this.handleClick);
   }
 
+  /**
+   * The first component mount will need to find the token item in the elastic search
+   * and then execute the data search, in case a token was already selected in the query
+   *
+   * @param {string} tokenId The token to pre select when first mounting the component
+   */
+  searchAndSelectToken = async (tokenId) => {
```

Comment:
`tokenId` is not being used. Either we remove this param or pass it on line 80 instead of `this.props.tokenId`

Reply:
Yeah, I was using this parameter in the beginning, then stopped using and forgot to remove. Thanks.

https://github.com/HathorNetwork/hathor-explorer/pull/201/commits/7c69c887c1f6f8c6e666f5e27b366dce13748cbe